src/main/msc/usbd_storage_sd_spi.c

```diff
@@ -216,7 +216,7 @@ static int8_t STORAGE_Read (uint8_t lun,
 {
 	UNUSED(lun);
 	for (int i = 0; i < blk_len; i++) {
-	    while (sdcard_readBlock(blk_addr + i, buf + (512 * i), NULL, NULL) == 0);
+		while (sdcard_readBlock(blk_addr + i, buf + (512 * i), NULL, 0) == 0);
```

Comment:
👾

Reply:
Any suggested change? I don't understand what you mean by that emoji.

Reply:
That means there are some unnecessary white spaces.